editor/store/defaults.js

```diff
@@ -81,4 +81,10 @@ export const EDITOR_SETTINGS_DEFAULTS = {
 
 	// Allowed block types for the editor, defaulting to true (all supported).
 	allowedBlockTypes: true,
+
+	// Maximum upload size in bytes allowed for the site.
+	maxUploadFileSize: 0,
```

Comment:
I don't know that long-term we want these to be considered editor settings, since they apply to the site as a whole. Seems like something which should ideally live in `@wordpress/core-data`, though I do not believe they are yet available from the REST API.